src/AvroHelper.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  AvroSchema,
+  RawAvroSchema,
+  AvroOptions,
+  ConfluentSchema,
+  SchemaHelper,
+  ConfluentSubject,
+} from './@types'
+import { ConfluentSchemaRegistryArgumentError } from './errors'
+import avro from 'avsc'
+
+export default class AvroHelper implements SchemaHelper {
+  private getRawAvroSchema(schema: ConfluentSchema): RawAvroSchema {
+    return JSON.parse(schema.schemaString) as RawAvroSchema
+  }
+
+  public getAvroSchema(schema: ConfluentSchema, opts?: AvroOptions) {
+    const rawSchema: RawAvroSchema = this.getRawAvroSchema(schema)
+    // @ts-ignore TODO: Fix typings for Schema...
+    const avroSchema: AvroSchema = avro.Type.forSchema(rawSchema, opts)
+    return avroSchema
+  }
+
+  public validate(avroSchema: AvroSchema): void {
+    if (!avroSchema.name) {
+      throw new ConfluentSchemaRegistryArgumentError(`Invalid name: ${avroSchema.name}`)
+    }
+  }
+
+  public getSubject(
+    schema: ConfluentSchema,
+    // @ts-ignore
```

Comment:
How come this is ts-ignored?

Reply:
because I thought `schema` would be useful to have here if we want it for the implementation of the other types, but the Avro implementation doesn't use this param so TS complains about it